test/libsolidity/InlineAssembly.cpp

```diff
@@ -246,7 +246,7 @@ BOOST_AUTO_TEST_CASE(functional)
 
 BOOST_AUTO_TEST_CASE(functional_partial)
 {
-	CHECK_PARSE_ERROR("{ let x := byte }", ParserError, "Expected token \"(\"");
+	CHECK_PARSE_ERROR("{ let x := byte }", ParserError, "Expected '(' (\"byte\" expects 2 arguments)");
```

Comment:
Perhaps not in scope for this PR, but we should change this to `(function \"byte\" expects 2 arguments)`